prince-of-versions/src/main/java/co/infinum/princeofversions/Exceptions.java

```diff
@@ -0,0 +1,24 @@
+package co.infinum.princeofversions;
+
+public interface Exceptions {
```

Comment:
I had an idea to have specific exception because lib throws it in case of some error in configuration. But I agree we could use any exception from java

Reply:
That makes sense. But then we should have a _base POV_ exception which specific exceptions will extend. If there is point of doing so, of course.